README.md

```diff
@@ -26,15 +26,18 @@ pip install devolearn
 ```
 ### Example notebooks
 <code>
-<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/nodes_matrix_long_smooth.gif" width = "40%">
-<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/3d_node_map.gif" width = "30%">  
+<p align="center">
+<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/nodes_matrix_long_smooth.gif" width = "40%" height="200">
+<img src = "https://raw.githubusercontent.com/DevoLearn/data-science-demos/master/Networks/3d_node_map.gif" width = "40%" height="200">  
+</p>
 </code>
 
```

Comment:
Please remove the `height` argument, it breaks the aspect ratio of the GIF when resizing the window

Reply:
Ok on it!

Reply:
Please mention the issue on the commit  message as:

`readme: fix #30: align images`